app/models/job_proxy_dispatcher.rb

```diff
@@ -93,6 +94,10 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def container_image_classes
+    ContainerImage.descendants.append(ContainerImage)
```

Comment:
I know the current pattern does this but it feels like the dispatcher shouldn't have this knowledge.  Additionally, it would cause us to load each of these classes and any requires they do in the schedule worker... a process that probably shouldn't load container dependencies.  I'm not sure of a better solution but if you have an idea, do let us know.